flasc/model_fit/model_fit.py

```diff
@@ -72,11 +72,12 @@ def __init__(
 
         # Get the number of turbines and confirm that
         # the dataframe and floris model have the same number of turbines
-        self.n_turbines = dfm.get_num_turbines(self.df)
```

Comment:
@paulf81 , I removed `n_turbines` from being an attribute of `ModelFit` here because I saw that it wasn't being used anywhere else in the class. However, this causes a test to fail. Do you see `n_turbines` as being a critical attribute of the `ModelFit` class? If so, I can add it back